docs/index.md

```diff
@@ -6,9 +6,12 @@ template: index.html
 
 <div class="grid-container" markdown>
 <div class="grid-child grid-child-sized" markdown>
-## **Getting Started**
 
-[DisGo](https://github.com/disgoorg/disgo) is a [Discord](https://discord.com/) API wrapper written in [Golang](https://golang.org/) aimed to be consistent, modular, customizable and less pain to use than other libraries.
+## **More information**
+Documentation is still under construction so you may consider also looking at these:
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/disgoorg/disgo.svg)](https://pkg.go.dev/github.com/disgoorg/disgo) <br>
+[![Discord Documentation](https://img.shields.io/badge/Discord%20Documentation-blue.svg)](https://discord.com/developers/docs)
```

Comment:
while I don't dislike these changes can you partially revert these?
the new info isn't bad